src/libraries/AccountAccessParser.sol

```diff
@@ -402,26 +402,43 @@ library AccountAccessParser {
         view
         returns (bool)
     {
-        if (isLivenessGuard(_account)) {
-            address[] memory owners = IGnosisSafe(_parentMultisig).getOwners();
-            for (uint256 i = 0; i < owners.length; i++) {
-                if (isGnosisSafe(owners[i])) {
-                    // Get the owners of the nested Gnosis Safe.
-                    address[] memory nestedSafeOwners = IGnosisSafe(owners[i]).getOwners();
-                    for (uint256 j = 0; j < nestedSafeOwners.length; j++) {
-                        // Check if the slot matches any liveness guard timestamp for the nested safe owners.
-                        bytes32 ownerSlot =
-                            keccak256(abi.encode(address(nestedSafeOwners[j]), LIVENESS_GUARD_LAST_LIVE_SLOT));
-                        if (_diff.slot == ownerSlot) {
-                            return true;
-                        }
+        if (!isLivenessGuard(_account)) {
+            return false;
+        }
+
+        return _checkOwnersForLivenessSlot(_parentMultisig, _diff.slot);
+    }
+
+    /// @notice Checks if the given slot matches any liveness guard timestamp for owners and nested safe owners
+    function _checkOwnersForLivenessSlot(address _multisig, bytes32 _slot) internal view returns (bool) {
```

Comment:
This is the first time we're executing a transaction directly from the security council safe. The normalize hash functionality wasn't picking up the liveness timestamps in this case. These changes make sure they do.